src/cmark.h

```diff
@@ -4,6 +4,7 @@
 #include <stdio.h>
 #include <cmark_export.h>
 #include <cmark_version.h>
+#include "memory.h"
```

Comment:
This line should be removed since `memory.h` is gone. It didn't break because there is a system header `memory.h`. Or did you mean to include the system header?